### PR TITLE
Add POPs to Trunks

### PR DESCRIPTION
**Pull request by korobey**
- [x] Add Pop as relationship to Trunk
- [x] Remove old trunk.preferred_server property
- [x] Add max_transfers and max_30x_redirects to sip_configurations

### CODE REVIEW

File: lib/didww/client.rb [hunk 1]
```diff
@@ -37,6 +37,10 @@ def balance
         Resource::Balance.all.first
       end
 
+      def cdr_exports
+        Resource::CdrExport
```
Comment (Fivell) [inline lib/didww/client.rb]: @korobey  I think it was changed by mistake, have nothing in common with Pops

Comment (korobey) [reply lib/didww/client.rb]: Indeed, just moved this to maintain alphabetical order of resources.

Comment (Fivell) [reply lib/didww/client.rb]: @korobey  such changes are better to apply via separate commit( to respect git history )

Comment (korobey) [reply lib/didww/client.rb]: @Fivell rebased

**Pull request merged**